terraform/variables.tf

```diff
@@ -1,3 +1,8 @@
+variable "region" {
```

Comment:
Can you add this to the `terraform.tfvars.example` file?

Reply:
done!